convert-policy.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
+# -*- coding: UTF-8 -*-
```

Comment:
Is this necessary still? It seemed surprising to me that it was as we don't seem to use any Unicode literals in the source text...

Reply:
It's not needed, but seems better to declare?

Reply:
Hmm, I don't know the best practices, but it feels like a workaround.

https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python-3 says it's not needed because the default is UTF-8, at least in Python 3.